coalib/output/ConsoleInteraction.py

```diff
@@ -597,7 +597,12 @@ def acquire_settings(log_printer, settings_names_dict, section):
     for setting_name, arr in sorted(settings_names_dict.items(),
                                     key=lambda x: (join_names(x[1][1:]), x[0])):
         value = require_setting(setting_name, arr, section)
-        result.update({setting_name: value} if value is not None else {})
+
+        while str(value).strip() == '':
+            logging.warning(str(setting_name) + ' setting is required.')
```

Comment:
`use_space setting is required` <--- Statement becomes confusing like asking setting of `use_space` 
I may be wrong here.

Reply:
Logic and test seems :+1:

Reply:
@Makman2 had a similar comment on the last PR
https://github.com/coala/coala/pull/3616#pullrequestreview-18992475

Have you tested it @RaiVaibhav

Reply:
>Have you tested it
Are you talking about the rest of above changes then no I didn't tested thats why I added a `comment` here not `requested any change`,the only problem I find after reading the code was the code was a message in `logging`